doc/model/axes/axis-cartesian-continuous.xml

```diff
@@ -67,20 +67,27 @@
             </c:documentation>
         </c:property>
 
-        <c:property name="ticksRegularity"
-                    type="pvc.options.varia.AxisTicksRegularity"
-                    default="'false'"
-                    category="Continuous-Numeric > Style">
+        <c:property name="preserveTickAlignment" 
+                    type="boolean"
+                    category="Continuous > Scale">
+            
             <c:documentation>
-                The axis' domain ticks regularity (applies to continuous axes).
+                When using a sliding window for real time applications,
+                the ticks alignment is used to guarantee that the ticks keep the 
+                same regularity from one rendering to the next (applies to continuous axes).
 
-                This actually depends on the value of <c:link to="#domainRoundMode" />:
-                <ul>
-                    <li>
-                        <p><b><i>domainRoundMode</i> should be <c:link to="pvc.options.varia.AxisDomainRoundingMode#None" /></b>
-                        in order to keep the scale domain exactly the domain of the data.</p>
-                    </li>
-                </ul>
+                This option works best with a <c:link to="pvc.options.varia.AxisDomainRoundingMode#None" /> 
+                value of <tt>none</tt>:
+
+                When <tt>true</tt>, the ticks alignment will be kept by preserving last minium
```

Comment:
Maybe:
`When <tt>true</tt>, it is ensured that the ticks of each new render are aligned with the ticks of the previous render, sharing values if possible.`
Also, please leave dates out of the description.